go.mod

```diff
@@ -2,13 +2,14 @@ module github.com/hashicorp/consul
 
 go 1.19
 
-replace github.com/hashicorp/consul/api => ./api
-
-replace github.com/hashicorp/consul/sdk => ./sdk
-
-replace github.com/hashicorp/consul/proto-public => ./proto-public
+replace (
+	github.com/hashicorp/consul/api => ./api
+	github.com/hashicorp/consul/proto-public => ./proto-public
+	github.com/hashicorp/consul/sdk => ./sdk
+)
 
-replace launchpad.net/gocheck => github.com/go-check/check v0.0.0-20140225173054-eb6ee6f84d0a
```

Comment:
I couldn't find any reference of this (even in our enterprise repo) so it feels safe to remove